assets/css/modules/_layout.scss

```diff
@@ -27,12 +27,16 @@
     border: 1px solid $border-grey;
   }
 }
+
+.main-container {
+  width: 100%;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
We should use @extend flex-column instead. Or at least add the appropriate vendor-prefixes to these, otherwise there will probably be cross browser problems.